dagrunner/tests/utils/test_function_to_parse.py

```diff
@@ -12,6 +12,24 @@
 CALLING_MOD = os.path.basename(sys.argv[0])
 
 
+def assert_help_str(help_str, tar):
+    # Remove line wraps for easier comparison
```

Comment:
Line wrap changes between versions of argparse, so we remove them.  Again a cause of failures on CI with success locally.